{{cookiecutter.project_slug}}/clients/react/src/utils/get-cookie.js

```diff
@@ -7,17 +7,17 @@
  *   const csrfToken = getCookie("csrftoken");
  */
 export default function getCookie(cname) {
-  var name = cname + "=";
-  var decodedCookie = decodeURIComponent(document.cookie);
-  var ca = decodedCookie.split(";");
+  var name = cname + '='
+  var decodedCookie = decodeURIComponent(document.cookie)
+  var ca = decodedCookie.split(';')
   for (var i = 0; i < ca.length; i++) {
-    var c = ca[i];
-    while (c.charAt(0) === " ") {
-      c = c.substring(1);
+    var c = ca[i]
+    while (c.charAt(0) === ' ') {
+      c = c.substring(1)
     }
     if (c.indexOf(name) === 0) {
-      return c.substring(name.length, c.length);
+      return c.substring(name.length, c.length)
     }
   }
-  return "";
+  return ''
```

Comment:
Unless there is a particularly strong reason to drop double questions and trailing semi-colons, I'd say we should NOT be doing that in this PR. I've seen several of our projects ping-pong back and forth between the two. We should pick one and stick with it.

Reply:
We're adding prettier and eslint in here as well.
I used the same prettier configuration we're using in the vue app:
https://github.com/thinknimble/tn-spa-bootstrapper/blob/edc1627c54162f05fb21d2f6d409893a9919df15/%7B%7Bcookiecutter.project_slug%7D%7D/clients/vue3/package.json#L70-L75